### ISSUE TITLE
et command

### ISSUE THREAD

**Issue by ZexMaxwell001** [Maselkov/GW2Bot-Red#10]
Is there plans to show HOT meta events in the et command? only the bosses are showing.

http://imgur.com/a/eFKwM

**Comment by Maselkov** [comment]
Yes, and hot phases are in files already as well. I forgot about it to be honest.
